README.md

```diff
@@ -78,6 +78,8 @@ gcloud:
     - /sdcard/screenshots
   performance-metrics: true
   test-targets:
+    # supported are class, notClass, package, notPackage, annotation, notAnnotation, testfile, notTestfile, size
+    # as described in https://developer.android.com/reference/android/support/test/runner/AndroidJUnitRunner
```

Comment:
I was wondering whether it would be nicer to actually support the available options directly through yaml like:
```
test-targets:
  class: ...
  notClass: ...
  package: ...
  notPackage: ...
  annotation: ...
  notAnnotation: ...
  testfile: ...
  notTestfile: ...
  size: ...
```
and then either pass the params as a list or just a comma separated string like:
```
test-targets:
  package: 
  - com.foo.bar
  size: 
  - large
```

Reply:
@winterDroid we're trying to match test-targets behavior of gcloud CLI. If they support that, great. If not then it's easier to maintain & debug flank if we match the same behavior.

Reply:
That makes sense. Then I guess we should keep it as is